src/components/v2/V2Project/V2ProjectReconfigureModal/hooks/reconfigureFundingCycle.ts

```diff
@@ -49,9 +51,26 @@ export const useReconfigureFundingCycle = ({
         }
       : editingFundingCycleMetadata
 
+    const weightBeforeReconfig = fundingCycle?.weight
+
+    // If weight is 0, we send weight=1 to the contract
+    // If weight has not been modified, we send weight=0 to the contract
+    let weightAfterReconfig = editingFundingCycleData.weight
```

Comment:
**suggestion** WDYT about moving this block of code to a function?

Call it `getNewFundingCycleWeight` or something.

Reply:
Good call, done